Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -240,8 +242,10 @@ class MessageQueue {
   __invokeCallback(cbID: number, args: Array<any>) {
     this._lastFlush = new Date().getTime();
     this._eventLoopStartTime = this._lastFlush;
+
+    //The rightmost bit of cbID indicates fail (0) or success (1), the other bits are the callID shifted left.
     const callID = cbID >> 1;
```

Comment:
In practice it usually won't matter but for the sake of correctness can you change this to `>>>`?